intro-to-rails/tasklist.checkpoint.md

```diff
@@ -0,0 +1,53 @@
+# TaskList
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=5436d2cc-09e9-4aed-8da2-ac5b0022db7d&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: short-answer
+* id: ead1577c-abd6-414b-a7a6-582176294a7d
+* title: TaskList Pull Request
+* points: 3
+* topics: rails
+
+##### !question
+
+Please fork and clone the [Ada-C14 Tasklist Repo](https://github.com/Ada-C14/task-list).
+
+Then cd into the folder with `cd tasklist`
+
+Then create a new rails project with `rails new .`
+
+Then complete the project and create a pull request. 
+
+When you create the pull request your tests will automatically run and you can see the results at the bottom of the PR.
+
+![Pull Request](images/automatic-tests.png)
+
+Please submit a link to your pull request here.
+
+##### !end-question
+
+##### !placeholder
+
+PR Link
+
+##### !end-placeholder
+
+##### !answer
+
+/^https:\/\/github\.com\/[^\/]+\/pull\/[^\/]+\/\d?\d/
```

Comment:
This regex expression doesn't match on `https://github.com/gSchool/ada-2-ruby-on-rails/pull/7` for me.

I think it's because you're missing a section of `\/[^\/]+` for organization AND repo name

This expression works for the following two test cases for me (including the trailing slash)
```
^https:\/\/github\.com\/[^\/]+\/[^\/]+\/pull\/\d?\d[\/]?
```

```
https://github.com/gSchool/ada-2-ruby-on-rails/pull/7

https://github.com/gSchool/ada-2-ruby-on-rails/pull/7/
```

Reply:
Thanks, I'd copied the regex from a previous project... updated and tested and it seems to work now.